sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/implementation/VirtualMachineUpdateTests.java

```diff
@@ -38,13 +38,12 @@ protected void cleanUpResources() {
     }
 
     @Test
-    @DoNotRecord(skipInPlayback = true)
     public void testVirtualMachineUpdate() {
         // Management Long running operation is Failed or Cancelled
         final String vmname = "javavm1";
 
-        final String mySqlInstallScript = "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/4397e808d07df60ff3cdfd1ae40999f0130eb1b3/mysql-standalone-server-ubuntu/scripts/install_mysql_server_5.6.sh";
-        final String installCommand = "bash install_mysql_server_5.6.sh Abc.123x(";
+        final String mySqlInstallScript = "https://raw.githubusercontent.com/Azure/azure-sdk-for-java/0f875b569cb30b3cd4232bd20abb02452431b1ad/sdk/resourcemanager/azure-resourcemanager-compute/src/test/assets/install_mysql_server_5.7.sh";
```

Comment:
use commit id `00df5c3ae1e25c526e265e78a00211d068b94f93` in path (it is on main branch)

Reply:
Fixed in the new version.